pkg/operation/cloudbotanist/azurebotanist/addons.go

```diff
@@ -68,5 +68,11 @@ func (b *AzureBotanist) GenerateAdmissionControlConfig() (map[string]interface{}
 
 // GenerateNginxIngressConfig generates values which are required to render the chart nginx-ingress properly.
 func (b *AzureBotanist) GenerateNginxIngressConfig() (map[string]interface{}, error) {
-	return common.GenerateAddonConfig(nil, b.Shoot.NginxIngressEnabled()), nil
+	return common.GenerateAddonConfig(map[string]interface{}{
```

Comment:
As transporting the `loadBalancerSourceRanges` to the Helm chart seems to be not cloud specific, please move this part to https://github.com/gardener/gardener/blob/master/pkg/operation/hybridbotanist/addons.go#L147 and keep the cloud specific implementations untouched.